users/tests/snapshots/snap_test_notifications.py

```diff
@@ -4,68 +4,68 @@
-] = [
-    """kukkuu@example.com|['new.email@example.com']|Guardian email change verification token requested FI|Guardian FI: Michael Patton (old.email@example.com).
+snapshots['test_guardian_change_email_token_requested_notification[new.email@example.com] 1'] = [
+    '''kukkuu@example.com|['new.email@example.com']|Guardian email change verification token requested FI|Guardian FI: Michael Patton (old.email@example.com).
         Token: abc123+-.
         Unsubscribe: http://localhost:3000/fi/profile/subscriptions?authToken=a1b2c3d4e5f6g7h8
-        """
+        '''
 ]
 
-snapshots[
-    "test_guardian_change_email_token_requested_notification[old.email@example.com] 1"
-] = [
-    """kukkuu@example.com|['old.email@example.com']|Guardian email change verification token requested FI|Guardian FI: Michael Patton (old.email@example.com).
+snapshots['test_guardian_change_email_token_requested_notification[old.email@example.com] 1'] = [
+    '''kukkuu@example.com|['old.email@example.com']|Guardian email change verification token requested FI|Guardian FI: Michael Patton (old.email@example.com).
         Token: abc123+-.
         Unsubscribe: http://localhost:3000/fi/profile/subscriptions?authToken=a1b2c3d4e5f6g7h8
-        """
+        '''
 ]
 
-snapshots["test_guardian_changed_email_notification[None] 1"] = []
+snapshots['test_guardian_changed_email_notification[None] 1'] = [
+]
 
-snapshots["test_guardian_changed_email_notification[new.email@example.com] 1"] = [
+snapshots['test_guardian_changed_email_notification[new.email@example.com] 1'] = [
     "kukkuu@example.com|['new.email@example.com']|Guardian email changed FI|Guardian FI: Michael Patton (new.email@example.com). Unsubscribe: http://localhost:3000/fi/profile/subscriptions?authToken=a1b2c3d4e5f6g7h8"
 ]
 
-snapshots["test_guardian_changed_email_notification[old.email@example.com] 1"] = []
+snapshots['test_guardian_changed_email_notification[old.email@example.com] 1'] = [
+]
 
-snapshots["test_send_user_auth_service_is_changing_with_children 1"] = [
-    """kukkuu@example.com|['michellewalker@example.net']|User authorization service is changing FI|
+snapshots['test_send_user_auth_service_is_changing_with_children 1'] = [
+    '''kukkuu@example.com|['michellewalker@example.net']|User authorization service is changing FI|
 Guardian FI: Michael Patton (michellewalker@example.net).
 The change is happening 17.6.2024.
 Childrens' event participation history:
 
 Child name: Jason Berg
 
 Event: Policy sport available.
-Occurrence: 1983-12-14 10:41:44.139987+00:00
+Occurrence: 1983-12-14 10:21:44.139987+00:00
 
 Event: Month score father middle brother station physical very.
-Occurrence: 1989-06-15 22:58:14.498478+00:00
+Occurrence: 1989-06-15 21:38:14.498478+00:00
 
 Child name: Katherine Gomez
 
 Event: Care any concern bed agree. Laugh prevent make never.
-Occurrence: 1970-12-08 12:46:35.206162+00:00
+Occurrence: 1970-12-08 12:26:35.206162+00:00
```

Comment:
Why does the snapshot now have 20 minutes difference to old times?

Reply:
Details in commit fc8e61820345a0ebeffe927de8560638725f6e32 message